### PR TITLE
Created PostgresProxy and !like command

### LINKED ISSUE #4: Utilize python-dotenv modue
https://pypi.org/project/python-dotenv/

### LINKED ISSUE #25: Create Postgres Proxy

### PR DESCRIPTION
**Pull request by camcast3**
Now as youtube_bot is running if a user sends the command:```!like``` it  will increment one value to the current song -> song_info and we will store the like into song_likes. 

The return to the user is "Thanks this data will help us continue bringing the best music to Prosolis Radio"**

### CODE REVIEW

File: bots/helpers/live_chat.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                    pageToken=next_page_token
+                )
+                response = request.execute()
+                logger.debug("Get LiveChatMessage NextPage list - Success!")
+
+            if response:
+                await discover_command_requests(youtube, response, live_chatid)
+
+            next_page_token = response["nextPageToken"]
+            wait_time = response["pollingIntervalMillis"]/1000
```
Comment (prosolis) [inline bots/helpers/live_chat.py:42]: Is Millis a common abbreviation for Millisecond (Msec)?

File: bots/helpers/live_chat.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    load_dotenv()
+    platform_name = 'Youtube'
+    postgres = postgresproxy.PostgresProxy(
+        user=os.getenv("POSTGRES_USER"), password=os.getenv("POSTGRES_PASSWORD"))
+    await postgres.song_likes_command(author_id, platform_name, fingerprint)
+
+    await mpd.mpd_connection_close()
+    await postgres.postgres_connection_close()
+
+    message = "Thanks this data will help us contiune bringing the best music to Radio Prosolis"
```
Comment (prosolis) [inline bots/helpers/live_chat.py]: 'Prosolis Radio' :)

**Review by prosolis — APPROVED** [review]

File: bots/helpers/live_chat.py (hunk 1 above)
Comment (camcast3) [reply bots/helpers/live_chat.py:42]: lol love how you put the more known abbreviation in the question. I have seen millsec but millis is weird.

pollingIntervalMillis is from the youtubeapi so we have no control over this name unfortunately.

File: bots/helpers/live_chat.py (hunk 2 above)
Comment (camcast3) [reply bots/helpers/live_chat.py]: 😲

**Pull request merged**